### PR TITLE
[fix](regression-test) Fix failed adaptive_pipeline_task_serial_read_on_limit

### PR DESCRIPTION
**Pull request by zhiqiang-hhhh**
1. Modify buckets from 10 to 5. The case will check whether `- MaxScannerThreadNum: 1` is contained, when buckets is 10, `- MaxScannerThreadNum: 10` will be misleading for the result prove.
2. `set adaptive_pipeline_task_serial_read_on_limit=20;` to make sure our strategy could work.

### CODE REVIEW

**Comment by zhiqiang-hhhh** [comment]
run buildall

File: regression-test/suites/query_profile/adaptive_pipeline_task_serial_read_on_limit.groovy
Comment (cjj2010) [inline regression-test/suites/query_profile/adaptive_pipeline_task_serial_read_on_limit.groovy:1]: LGTM

**Review by cjj2010 — APPROVED** [review]
LGTM

**Comment by zhiqiang-hhhh** [comment]
test failed on known problem

**Comment by zhiqiang-hhhh** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**